web/src/router.rs

```diff
@@ -210,10 +229,14 @@ mod organization_endpoints_tests {
             .send()
             .await?;
 
-        let response_text = response.text().await?;
+        // We need to parse the values to serde_json::Value to compare them
+        // // so that the attribute order does not matter.
```

Comment:
```suggestion
        // so that the attribute order does not matter.
```

Reply:
updated in 3d201b56bb1b97fe95447580f25b3ce9ba5afb7f!